reference/configuration/framework.rst

```diff
@@ -196,9 +197,35 @@ see :doc:`/cookbook/request/load_balancer_reverse_proxy`.
 form
 ~~~~
 
+.. _form-enabled:
+
+enabled
+.......
+
+**type**: ``boolean`` **default**: ``false``
+
+Whether or not to enable support for the Form component.
```

Comment:
see my comment below about a reason to disable the CSRF layer. And given that forms are enabling the validation layer by default too, this involves a lot more stuff in your container.